### PR TITLE
Prepare 0.25.3 release

### PR DESCRIPTION
**Pull request by mtreinish**
### Summary

This commit prepares the 0.25.3 release, this consists of bumping the version number to 0.25.3 in all the appropriate files. It also adds a prelude release note to provide a brief introduction to the release. When this commit merges it should be tagged as the 0.25.3 release.

### Details and comments

TODO:

- [x] Wait for final bugfixes to merge
- [x] Update `docs/release_notes.rst` with `reno report` output

### CODE REVIEW

**Comment by 1ucian0** [comment]
> * [ ]  Wait for final bugfixes to merge

This one seems done https://github.com/Qiskit/qiskit/milestone/46

**Comment by mtreinish** [comment]
Thanks, yeah the last PR merged overnight. I've updated the release notes so this should be good to review now.

**Review by jakelishman — DISMISSED** [review]
This seems right to me, though I'm feeling burned about the manual version handling haha.

File: docs/release_notes.rst [hunk 1]
```diff
@@ -126,6 +127,77 @@ Qiskit Metapackage Version  qiskit-terra  qiskit-aer  qiskit-ignis  qiskit-ibmq-
...
+
+- Fix the Quantum Shannon Decomposition implemented in :func:`.qs_decomposition`. When a unitary
+  could not take advantage of the diagonal commutation optimization, it used to error.
+  Now, it leaves it as undecomposed 2-qubit unitary gate.
+  Fixes `#10787 <https://github.com/Qiskit/qiskit/issues/10787>`__
+
+.. releasenotes/notes/backend-estimator-v2-support-a698353aeeb5236c.yaml @ b'af48b85af9f65236f4c4568a92c34a896b1157da'
+
+- Fixed a bug where :class:`.primitives.BackendEstimator` cannot be used with the
+  :class:`.providers.BackendV2` that does not have coupling_map.
```
Comment (1ucian0) [inline docs/release_notes.rst:163]:
```suggestion
  :class:`.providers.BackendV2` that does not have ``coupling_map``.
```

File: docs/release_notes.rst [hunk 2]
```diff
@@ -126,6 +127,77 @@ Qiskit Metapackage Version  qiskit-terra  qiskit-aer  qiskit-ignis  qiskit-ibmq-
...
+
+.. releasenotes/notes/fix-error-message-qpy-version-cf0763da22ce2224.yaml @ b'aca76ccc876e983e26a36b5d9369a152781e6d8c'
+
+- Fixed an issue with :func:`.qpy.load` when attempting to load a QPY format
+  version that is not supported by this version of Qiskit it will now display
+  a descriptive error message. Previously, it would raise an internal error
+  because of the incompatibility between the formats which was difficult to
+  debug. If the QPY format verison is not supported that indicates the Qiskit
```
Comment (1ucian0) [inline docs/release_notes.rst:171]:
```suggestion
  debug. If the QPY format version is not supported that indicates the Qiskit
```

File: docs/release_notes.rst [hunk 3]
```diff
@@ -126,6 +127,77 @@ Qiskit Metapackage Version  qiskit-terra  qiskit-aer  qiskit-ignis  qiskit-ibmq-
...
+
+- Fixed a regression in :func:`.transpile`, where an ``initial_layout`` given
+  as a :class:`range` object would no longer be treated as a valid input.
+  Fixed `#10544 <https://github.com/Qiskit/qiskit/issues/10554>`__.
+
+.. releasenotes/notes/fix_backend_name-e84661707058b529.yaml @ b'c3b652cd7310c240297a0037421fca312f0df124'
+
+- Fixed an issue in the :class:`.QuantumInstance` class where it was assuming
+  all ``AerSimulator`` backends were always :class:`.BackendV1`. This would cause
+  combatibility issues with the 0.13.0 release of ``qiskit-aer`` which is starting to
```
Comment (1ucian0) [inline docs/release_notes.rst:193]:
```suggestion
  compatibility issues with the 0.13.0 release of ``qiskit-aer`` which is starting to
```

**Review by 1ucian0 — DISMISSED** [review]
LGTM, just some small typos in release notes. They might not be worthy to block on that.

File: docs/release_notes.rst (hunk 1 above)
Comment (mtreinish) [reply docs/release_notes.rst:163]: I'll actually do this as

```suggestion
  :class:`.providers.BackendV2` that does not have any connectivity constraints.
```

I guess we could say :attr:`.BackendV2.coupling_map` is ``None`` too. But I think lets keep it higher level.

Comment (mtreinish) [reply docs/release_notes.rst:163]: Done in: https://github.com/Qiskit/qiskit/pull/11102/commits/213676492c50e4df16e9f3437d4ee9fb911f8955 (I forgot the `release_notes.rst` file is too large and github won't apply suggestions).

File: docs/release_notes.rst (hunk 2 above)
Comment (mtreinish) [reply docs/release_notes.rst:171]: Done in: https://github.com/Qiskit/qiskit/pull/11102/commits/213676492c50e4df16e9f3437d4ee9fb911f8955

File: docs/release_notes.rst (hunk 3 above)
Comment (mtreinish) [reply docs/release_notes.rst:193]: Done in: https://github.com/Qiskit/qiskit/pull/11102/commits/213676492c50e4df16e9f3437d4ee9fb911f8955

**Review by jakelishman — APPROVED** [review]

**Pull request merged**